www/explore/teams/index.spt

```diff
@@ -0,0 +1,43 @@
+[-----------------------------]
+
+teams = website.db.all("""
+
+    SELECT t.*
+         , p.receiving
+         , p.receiving / t.nmembers AS dollars_per_member 
+      FROM ( SELECT team            AS name
+                  , count(member)   AS nmembers
+               FROM current_takes
+           GROUP BY team
+            ) AS t
+      JOIN participants p 
+        ON p.username = t.name
+  ORDER BY dollars_per_member DESC, nmembers ASC
```

Comment:
Done in 1c3bc7c77c749b1068828706a4bac3471dd62bc7. I also added `goal` and `% funded` columns:

![screen shot 2015-02-13 at 4 00 27 pm](https://cloud.githubusercontent.com/assets/134455/6195215/00d86556-b39a-11e4-8538-05745d78ba8c.png)